spec/integration/standard_tasks_spec.rb

```diff
@@ -58,13 +58,13 @@
     it 'converts odt' do
       content = fixture('heathen/ooword.odt').read
       new_content = converter.convert 'txt', content
-      expect(new_content.mime_type).to eq 'text/plain; charset=utf-8'
+      expect(new_content.mime_type).to eq 'text/plain; charset=us-ascii'
     end
 
     it 'converts docx' do
       content = fixture('heathen/msword.docx').read
       new_content = converter.convert 'txt', content
-      expect(new_content.mime_type).to eq 'text/plain; charset=utf-8'
+      expect(new_content.mime_type).to eq 'text/plain; charset=us-ascii'
```

Comment:
totally agreed, when I come back to rome I'd like to show you a problem in the local environment with the charset that breaks my tmux session.